src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -88,6 +106,18 @@ export const RequestCodeViewer = ({ indexPattern, json }: RequestCodeViewerProps
               {openInConsoleLabel}
             </EuiButtonEmpty>
           )}
+          {shouldShowsearchProfilerLink && (
+            <EuiButtonEmpty
+              size="xs"
+              flush="right"
+              iconType="search"
+              href={searchProfilerHref}
+              onClick={handlesearchProfilerLinkClick}
```

Comment:
@sabarasaba and I were just discussing that using both `href` and `onClick` simultaneously on a button is not recommended (https://github.com/elastic/kibana/pull/127156/files#r828186351). What do you think about using `RedirectAppLinks` for correct routing here instead?

Reply:
This is a great point! I opened up a separate issue to address this, as the same problem exists for the "Open in Console" link - https://github.com/elastic/kibana/issues/128180